eventually/tests/unittests/authentication/testview.py

```diff
@@ -23,6 +26,33 @@ def setUp(self):
         not_active_user.set_password('password')
         not_active_user.save()
 
+        time_mock = datetime.datetime(2017, 4, 10, 12, 00, tzinfo=pytz.utc)
+        with mock.patch('django.utils.timezone.now') as mock_time:
+            mock_time.return_value = time_mock
+
+            self.client = Client()
+
+            user = CustomUser.objects.create(id=100,
```

Comment:
is it possible to create only one user for our test cases to optimise purposes? =)

Reply:
One user removed, another two are necessary for delete user method.